include/robot_state_publisher/joint_state_listener.h

```diff
@@ -65,15 +65,15 @@ class JointStateListener {
    */
   JointStateListener(const KDL::Tree& tree, const MimicMap& m, const urdf::Model& model = urdf::Model());
 
-  JointStateListener(const std::shared_ptr<RobotStatePublisher>& rsp, const MimicMap& m);
+  JointStateListener(std::shared_ptr<RobotStatePublisher>  rsp, MimicMap  m);
```

Comment:
I agree with the change to the `rsp` shared_ptr, but not with the change to the `MimicMap`.  That will cause a copy of the map to be made, which isn't what we want.

Reply:
Take a look at the `mimic_` member variable. We initialize using move semantics now.